test/lifecycle_test.dart

```diff
@@ -673,6 +676,46 @@ void sharedLifecycleTests<T extends react.Component>({
           ].where((matcher) => matcher != null).toList()));
     });
 
+    if (isComponent2) {
+      test('init testing', () {
```

Comment:
This name could be more descriptive
```suggestion
      test('initializes state with the value set in `init`', () {
```